src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -3703,12 +3703,17 @@ def list_vmss_instance_public_ips(cmd, resource_group_name, vm_scale_set_name):
     return [r for r in result if r.ip_address]
 
 
-def reimage_vmss(cmd, resource_group_name, vm_scale_set_name, instance_id=None, no_wait=False):
+def reimage_vmss(cmd, resource_group_name, vm_scale_set_name, instance_id=None, instance_ids=None, no_wait=False):
```

Comment:
Retaining both `instance_id` and `instance_ids` makes the code hard to maintain.

Reply:
Updated